lib/md2html.js

```diff
@@ -10,18 +10,20 @@ function enddetails(l, content) {
 }
 
 filter.stdio(function ({ t, c }, format, meta) {
-  if (
-    t === "Header" &&
-    /^(table-of-contents|appendix-[a-z]-|[a-z]?\d+-)/.test(c[1])
-  ) {
-    const content = [
-      filter.RawBlock("html", `<details open><summary>`),
-      filter.Header(...c),
-      filter.RawBlock("html", `</summary>`),
-    ];
-    enddetails(c[0], content);
-    level = c[0];
-    return content;
+  if (t === "Header") {
+    if (c[0] === 1 || /^(appendix-[a-z]-|[a-z]?\d+-)/.test(c[1][0])) {
+      const content = [
+        filter.RawBlock("html", `<details open><summary>`),
+        filter.Header(...c),
+        filter.RawBlock("html", `</summary>`),
+      ];
+      enddetails(c[0], content);
```

Comment:
Should also be called at the end of the document, but see https://stackoverflow.com/questions/78962777.